test/boulder-config.json

```diff
@@ -116,7 +116,8 @@
   },
 
   "pa": {
-    "dbConnect": "mysql+tcp://policy@localhost:3306/boulder_policy_integration"
+    "dbConnect": "mysql+tcp://policy@localhost:3306/boulder_policy_integration",
+    "challengeTypes": ["simpleHttp", "http-01", "tls-sni-01"]
```

Comment:
Not that it's a problem, but this means `boulder-ca` has to know about the `challengeTypes`, as it contains a `pa`. Is there extra value that can be derived there, somehow? (Extra sanity checks, for example?)